helpers/contentful_helpers.rb

```diff
@@ -19,7 +19,8 @@ module ContentfulHelpers
           'product' => yml[:product],
           'section' => 'Blog',
           'type' => yml[:type],
-          'hackernews_link' => yml[:hackernews_link]
+          'hackernews_link' => yml[:hackernews_link],
+          'show_body_in_index' => yml[:show_body_in_index]
```

Comment:
This change will require some updates to specs.

Reply:
Yeah; done. While I was in there, I also updated the specs so they run one example per fixture, as opposed to one example with all fixtures, which makes the error output on failing specs a lot more useful (because it tells you *which* fixture failed).

Reply:
👍 , thank you!